sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationEventProcessor/EventResult/DialogEventResult.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary><see cref="DialogEventResult"/> is returned from WaitForEvent of <see cref="DialogResult"/>.</summary>
+    public class DialogEventResult
+    {
+        /// <summary>
+        /// Indicates whether the returned event is considered successful or not.
+        /// </summary>
+        public bool IsSuccessEvent { get; internal set; }
+        /// <summary>
+        /// <see cref="DialogCompleted"/> event will be returned once the dialog is completed successfully.
+        /// </summary>
+        public DialogCompleted DialogCompletedSuccessEvent { get; }
```

Comment:
...SuccessResult.
See this page on how the naming is done: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationEventProcessor/EventResult/AddParticipantEventResult.cs